test/core/event_engine/iomgr_event_engine/event_poller_posix_test.cc

```diff
@@ -479,16 +480,20 @@ int main(int argc, char** argv) {
   ::testing::InitGoogleTest(&argc, argv);
   auto engine =
       absl::make_unique<grpc_event_engine::experimental::IomgrEventEngine>();
+  int result = 0;
   EXPECT_NE(engine, nullptr);
   grpc_event_engine::iomgr_engine::TestScheduler scheduler(engine.get());
-  g_event_poller =
-      grpc_event_engine::iomgr_engine::GetDefaultPoller(&scheduler);
-  if (g_event_poller == nullptr) {
-    // Poller is not supported on this system.
-    return 0;
+  // Try both pollers (epoll1 and poll) one after the other.
+  g_event_poller = grpc_event_engine::iomgr_engine::GetEpoll1Poller(&scheduler);
```

Comment:
Let's do a test fixture, and parameterize the test based on poller implementation?

Reply:
Done. I parametrized instead by poll strategy name to also test the code which creates the default poller to use.